docs/rancher/03-setup-rancher/disaster-recovery-rancher.md

```diff
@@ -0,0 +1,147 @@
+# Rancher Disaster Recovery
+
+## Introduction
+
+This document offers a detailed guide on how to restore your Rancher deployment to a new cluster if the original node running Rancher fails.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Step 1 - Switch to the k3s cluster](#step-1---switch-to-the-k3s-cluster)
+- [Step 2 - Install the rancher-backup Helm chart](#step-2---install-the-rancher-backup-helm-chart)
+- [Step 3 - Restore from backup using a Restore custom resource](#step-3---restore-from-backup-using-a-restore-custom-resource)
+- [Step 4 - Follow the Setting Up Rancher on k3s Guide](#step-4---follow-the-setting-up-rancher-on-k3s-guide)
+- [Conclusion](#conclusion)
+
+## Prerequisites
+
+Before beginning the restoration process, ensure you have:
+
+- A newly set up server following the [Setup Ubuntu](../01-setup-ubuntu/README.md) guide.
```

Comment:
We also need to setup k3s before restoring the backup.
https://github.com/MurmurationsNetwork/MurmurationsServices/blob/main/docs/rancher/02-setup-k3s/README.md